multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -91,6 +91,10 @@ def parse_genome_results(self, f):
         "mean_mapping_quality": r"mean mapping quality = ([\d,\.]+)",
         "general_error_rate": r"general error rate = ([\d,\.]+)",
         "mean_coverage": r"mean coverageData = ([\d,\.]+)",
+        "regions_size": r"regions size = ([\d,\.]+)",
+    }
+    regexes_secondary = {
+        "regions_mapped_reads": r"number of mapped reads = ([\d,]+)",
     }
```

Comment:
Why does this need to be in its own dict? Can it not just be added to the previous block?

Reply:
If I remember correctly, it's something like the way the loop was constructed for searching each regex, it didn't allow the pick up of the 'second' line with the string (there is nothing to distinguish from the earlier line, other than a 'section' header a few lines above the second hit of this line.

Reply:
But I'm going to have to refer to @TCLamnidis for the python details (he fixed most of this for me 😅)

Reply:
I suppose it would be possible to change the entire regex matching to `.findall` and set for all other values that we want the first element, then do a check for `len > 1` for that regex to separate the two values.
This approach seemed "safer" in that we are not changing the matching for values that work as intended. "Don't fix what ain't broken."